person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super(name)
```

Comment:
- [ ] This line causes an error because the parent class `Nameable` has no `initialize` method which is called by the `super`. So it will cause a `wrong number of arguments` error. Kindly fix this issue

### Hint:
You can remove this line or use super without any parameter.

Reply:
@gemmen29 moved this function correct_name inside of the person class and used the super without the parameter which is fixed now and running without any error, thankyou you for correcting me.